tiatoolbox/tools/stainnorm.py

```diff
@@ -118,16 +141,20 @@ def __init__(self, stain_matrix):
 
 
 class RuifrokNormaliser(StainNormaliser):
-    """Ruifrok stain normaliser, adapted from:
-    A.C. Ruifrok & D.A. Johnston 'Quantification of histochemical staining
-    by color deconvolution'. Analytical and quantitative cytology and histology
-    / the International Academy of Cytology and American Society of Cytology.
+    """Ruifrok & Johnston stain normaliser, adapted from:
```

Comment:
First line should end with a period.

"""Ruifrok & Johnston stain normaliser.
This code is adapted from: ....

"""

Reply:
Fixed